tests/utils/botocore/__init__.py

```diff
@@ -32,6 +31,7 @@
 import datetime
 from io import BytesIO
 from subprocess import Popen, PIPE
+from unittest import mock
```

Comment:
Probably a change we should do in the future, but most of the usage for `mock` is from the backport `mock` library. I'd prefer not updating it now to keep this change minimal. I think there is a larger effort to switch to using unittest's mock.

Reply:
Yep, this was a miss from testing. I think this work was already done for v1 last fall so it should fairly straight forward to port.